cli/Valet/Site.php

```diff
@@ -328,28 +328,43 @@ function pruneLinks()
     }
 
     /**
-     * Resecure all currently secured sites with a fresh tld.
+     * Resecure all currently secured sites with a fresh configuration.
      *
-     * @param  string  $oldTld
-     * @param  string  $tld
+     * @param  array  $old
+     * @param  array  $new
      * @return void
      */
-    function resecureForNewTld($oldTld, $tld)
+    function resecureForNewConfiguration(array $old, array $new)
     {
         if (! $this->files->exists($this->certificatesPath())) {
             return;
         }
 
         $secured = $this->secured();
 
+        $defaultTld = $this->config->read()['tld'];
+        $oldTld = isset($old['tld']) && ! empty($old['tld']) ? $old['tld'] : $defaultTld;
+        $tld = isset($new['tld']) && ! empty($new['tld']) ? $new['tld'] : $defaultTld;
```

Comment:
Can `isset() && !empty()` be replace with just `!empty()`, since we're never expecting `false` or `0` here?